docs/source/data-model.rst

```diff
@@ -20,31 +32,39 @@ JSON) in any language. Each document class has a defined, but flexible, schema.
 * Event Descriptor --- Metadata about a series of Events. Envision
   richly-detail column headings in a table, encompassing physical units,
   hardware configuration information, etc.
+* Resource --- A pointer to an external file (or resource in general).
+* Datum --- A pointer to a specific slice of data within a Resource.
 * Run Stop Document ---  Everything that we can only know at the very end, such
   as the time it ended and the exit status (succeeded, aborted, failed due to
   error).
 
+Every document has a ``time`` (its creation time) and a separate ``uid`` to
+identify it. The Event documents also have a ``descriptor`` field linking them
```

Comment:
When this was all new to me, the two terms "Event Descriptor" and "`descriptor`" as a document schema did not jump out as being the same thing.  *Run Start*/`start` and *Run Stop*/`stop` similar but somehow easier to grasp.  Gradually, this struck home.

Is this the time to make this crystal clear for the neophytes?  The new terminology (with familiar names) in bluesky is daunting to new users.

Reply:
It would be valuable to pick one term from each pair and stick with it throughout this documentation. My preferences are "Event Descriptor", "Start", and "Stop" for what it's worth. But I think a new PR should be opened for this change.

Reply:
I agree.  A new PR *after* this is all merged.